package.json

```diff
@@ -18,53 +19,60 @@
   "bugs": {
     "url": "https://github.com/e-Potek/file-viewer/issues"
   },
+  "files": [
+    "dist"
+  ],
   "homepage": "https://github.com/e-Potek/file-viewer#readme",
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/eslint-parser": "^7.17.0",
-    "@babel/plugin-transform-runtime": "^7.17.0",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-react": "^7.16.7",
-    "@emotion/react": "^11.9.0",
-    "@emotion/styled": "^11.8.1",
-    "@mui/material": "^5.6.4",
+    "@babel/core": "^7.18.10",
```

Comment:
If these are the latest versions of the packages, they might be higher than our own repo, which could result in installing 2 versions of the packages. Did you check this, or should I update our packages?

Reply:
I checked and it does not seem to affect it in any way, it works the same as before. Since these are dev dependencies, they should not be installed again when using this as a package.

Reply:
I checked and it was working, but I downgraded the react and @mui/material peerDependencies just to be safe, otherwise it might show warning on npm install. The other dependencies are devDependencies and they should not matter since they are not installed when the package is installed.